sample_configs/bootstrap.sh

```diff
@@ -16,6 +16,7 @@ apt-get install -y apache2 \
 	php5-dev \
 	php-pear \
 	php5-memcached \
+	php5-xdebug \
```

Comment:
I feel like you should be able to make use of bootstrap.sh (& vagrant?) as part of travis. Otherwise it's just going to get outdated (again) - I already note that tbootstrap.sh uses psql 9.3, vs travis using 9.4